### PR TITLE
solved #96 and #86

### PR DESCRIPTION
**Pull request by YashKumarVerma**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a live link or preview screenshot
- [ ] Images are `240 x 240` [w x h]
- [ ] Resolved merge conflicts
- [ ] Included a description of my change below

# Things done in this Pull Request

- Fixed issue #86 
- Fixed issue #96 

check preview at [https://yashkumarverma.github.io/gci17.fossasia.org/](https://yashkumarverma.github.io/gci17.fossasia.org/)

### CODE REVIEW

**Review by anshumanv — APPROVED** [review]

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+### Expected results
```
Comment (anshumanv) [inline .github/ISSUE_TEMPLATE.md:1]: I think Actual results should come before Expected, since that will describe the issue.

**Review by ParthS007 — APPROVED** [review]
LGTM !

**Comment by ParthS007** [comment]
@anshumanv I think we should merge this as no of PR is increasing and we need this in urgent !

**Comment by anshumanv** [comment]
Right, minor details can be corrected later and we need to shorten down the list. :smile: 
@ParthS007 merging.

**Pull request merged**